### PR TITLE
Send receive data from API

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
In this PR, I have:
- Implemented the refresh button to get the scores from the API
- Implemented the add button to add new scores.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,17 +2,19 @@ import './styles/main.scss';
...
 
-function addScoreToList(name, score) {
-  scores.push({ name, score });
-  scores.sort((a, b) => b.score - a.score);
+const gameId = 'p54b5mtGhUUyB50K6Ux1';
+const scores = [];
 
+// Create a new game with the name "My Game" just one time to get the gameId
+// and then use it to add scores and get the leaderboard.
+function renderScores() {
```
Comment (BregornOriginal) [inline src/index.js]: As we need to follow the new standards of ES6, it's necessary to start using `arrow functions`

- Kindly refactor this function method for an `arrow function`

File: src/index.js [hunk 2]
```diff
@@ -22,10 +24,46 @@ function addScoreToList(name, score) {
   });
 }
 
-addForm.addEventListener('submit', (event) => {
+async function getScores() {
```
Comment (BregornOriginal) [inline src/index.js]: As we need to follow the new standards of ES6, it's necessary to start using `arrow functions`

- Kindly refactor this function method for an `arrow function`

File: src/index.js [hunk 3]
```diff
@@ -22,10 +24,46 @@ function addScoreToList(name, score) {
...
+  const response = await fetch(`${apiUrl}/games/${gameId}/scores`);
+  const data = await response.json();
+  scores.length = 0;
+  data.result.forEach((score) => {
+    scores.push({ user: score.user, score: score.score });
+  });
+  renderScores();
+}
+
+async function addScore(name, score) {
```
Comment (BregornOriginal) [inline src/index.js]: As we need to follow the new standards of ES6, it's necessary to start using `arrow functions`

- Kindly refactor this function method for an `arrow function`

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

hand-sketching-stair-slogan-one-step-closer-chalkboard-79948193


## Highlights

✔️ Linters are passing
✔️ Good commit history
✔️ README looks professional
✔️ Good work implementing `async` and `await` with `fetch`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
oh! 😞 my bad! thank you for your review @BregornOriginal

**Review by DammyShittu — APPROVED** [review]
Hi @rachaouldbabaali,

## APPROVED 🟢

## Highlights
✔️ Arrow functions

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**